### ISSUE TITLE
Nomalization factor in kdeplot

### ISSUE THREAD

**Issue by Dimchord** [mwaskom/seaborn#1396]
I know this feature has been suggested before as part of #479. My use case is however a bit different.

I suggest to put a normalization factor in the `kdeplot` function. When looking at a subset of data, I would like to scale the KDE to normalize to the fraction of included data. Now you get something like the figure below, where blue is the total data set and green/orange are two subsets of my data. If we could include a normalization parameter, we can match the KDE's of the subsets to the KDE of the total set.

![kde_problem](https://user-images.githubusercontent.com/9883328/38370938-976a9348-38eb-11e8-8113-a87334a06350.png)

**Comment by Dimchord** [comment]
No, it is an entirely different issue. That issue is about setting the Axis limits.

**Comment by snoopyhunter2** [comment]
I am having the same issue, and it is not related to the issue #61. 

Please consider the following minimal example:

`import numpy as np
import seaborn as sns
import matplotlib.pyplot as plt

####################

data1 = np.random.rand(100)/100 + 1
data2 = np.random.rand(100)/100 - 1

tot_data = np.concatenate((data1, data2))

plt.figure()
sns.kdeplot(tot_data)
plt.title('only total data')

plt.figure()
sns.kdeplot(tot_data)
sns.kdeplot(data1)
sns.kdeplot(data2)
plt.title('total data with separate subsets')`

![fig1](https://user-images.githubusercontent.com/34123302/49160969-31876b80-f328-11e8-98f8-da83c01a8bd7.png)
![fig2](https://user-images.githubusercontent.com/34123302/49160976-364c1f80-f328-11e8-8dd3-ef60def11d0a.png)

Changing the normalisation factor would allow for a graph on which one could show qualitative features of both the total dataset and its subsets.

**Comment by mwaskom** [comment]
You don’t want to change the vertical normalization, you want to change the bandwidth. But also think hard about what you’re trying to accomplish by using a Gaussian kernel density estimate here.

**Comment by snoopyhunter2** [comment]
You are right of course, in a quantitative way of thinking it makes absolutely no sense to rescale a kde. 
However, apart from being used to quantitatively estimate probability density functions, showing the kde is also a handy way to highlight some qualitative properties of the data's distribution. 
For this second application, it is my feeling that "rescaling" the kde would be a useful feature here.

Of course, I could also play around with the bandwidth, thanks for that suggestion.
